github/apps_installation.go

```diff
@@ -10,14 +10,28 @@ import (
 	"fmt"
 )
 
+// InstallationPermissions Lists the permissions for metadata, contents, issues and single file for an installation.
+type InstallationPermissions struct {
+	MetaData   *string `json:"metadata,omitempty"`
```

Comment:
To be consistent with our naming policies, please change this field name from `MetaData` to `Metadata` since the corresponding JSON field is not `meta_data`.

Reply:
👍